curator/api/allocation.py

```diff
@@ -3,12 +3,13 @@
 import logging
 logger = logging.getLogger(__name__)
 
-def apply_allocation_rule(client, indices, rule=None):
+def apply_allocation_rule(client, indices, rule=None, allocation_type='require' ):
     """
     Apply a required allocation rule to a list of indices.
 
     :arg client: The Elasticsearch client connection
     :arg indices: A list of indices to act on
+    :arg allocation_type: Which type of allocation to apply
```

Comment:
Please move this below `rule`, so they appear in the same order the arguments appear in the definition.